R/settings_docsify.R

```diff
@@ -0,0 +1,93 @@
+.import_settings_docsify <- function(path) {
+    # Read settings sidebar
+    fn <- fs::path_join(c(path, "altdoc", "docsify.md"))
+    sidebar <- readLines(fn)
+
+    # Single files
+    if (fs::file_exists(fs::path_join(c(.doc_path(path), "NEWS.md")))) {
+        sidebar <- gsub("\\$ALTDOC_NEWS", "NEWS.md", sidebar)
+    } else {
+        sidebar <- gsub("\\$ALTDOC_NEWS", "", sidebar)
+    }
+
+    if (fs::file_exists(fs::path_join(c(.doc_path(path), "LICENSE.md")))) {
+        sidebar <- gsub("\\$ALTDOC_LICENSE", "LICENSE.md", sidebar)
+    } else {
+        sidebar <- gsub("\\$ALTDOC_LICENSE", "", sidebar)
+    }
+
+    if (fs::file_exists(fs::path_join(c(.doc_path(path), "CODE_OF_CONDUCT.md")))) {
+        sidebar <- gsub("\\$ALTDOC_CODE_OF_CONDUCT", "CODE_OF_CONDUCT.md", sidebar)
+    } else {
+        sidebar <- gsub("\\$ALTDOC_CODE_OF_CONDUCT", "", sidebar)
+    }
+
+    ############### Vignettes
+    # TODO: get clean titles. .get_vignettes_titles does not work as I expected
+    dn1 <- fs::path_join(c(.doc_path(path), "vignettes"))
```

Comment:
is there a folder "docs/vignettes" at some point? I thought all vignettes were moved to "docs/articles"

Reply:
I opened an issue about this.